src/Gir/Generation/IndentWriter.cs

```diff
@@ -28,6 +28,7 @@ public IndentWriter WriteHeader ()
 				return this;
 			
 			WriteLine ("using System;");
+			WriteLine ("using System.Runtime.InteropServices;");
```

Comment:
Do we need to add this to every file?

Reply:
In almost every file. All structs will need the LayoutKind attribute and we will need the Marshal class quite often. I think it's worth it to include it in any file rather than checking if we really need it. The old generator included it in any file as well.

Reply:
Agreed with @stsundermann